app/views/shared/_check_answers_vehicles.html.erb

```diff
@@ -0,0 +1,40 @@
+<h2 class="govuk-heading-m">
+  <%= t('.heading') %>
+</h2>
+
+<dl class="govuk-summary-list govuk-!-margin-bottom-9">
+  <%= check_answer_link(
+        name: :vehicles,
+        url: check_answer_url_for(user_type, :vehicles, application),
+        question: t('.own'),
+        answer: yes_no(application&.vehicle&.persisted?)
```

Comment:
`application&.vehicle&.persisted?` is used repeatedly in this file. If the code below was added to the `LegalAidApplication` model:

```ruby
delegate :persisted?, to: :vehicle, prefix: true, allow_nil: true
```

`application&.vehicle&.persisted?` could be replaced with `application.vehicle_persisted?`

Reply:
yes that's much better 👍